Lib/subprocess.py

```diff
@@ -1414,6 +1419,16 @@ def _execute_child(self, args, executable, preexec_fn, close_fds,
 
             if executable is None:
                 executable = args[0]
+
+            if (preexec_fn is None
+                    and not close_fds
+                    and not pass_fds
+                    and cwd is None
+                    and env is None
+                    and restore_signals is False
```

Comment:
"and not restore_signals" would be better. I prefer to avoid "is" to compare with True/False.